auto_nag/topcrash.py

```diff
@@ -361,10 +423,7 @@ def handler(search_resp: dict, data: dict):
 
                 name = signature["term"]
                 installations = signature["facets"]["cardinality_install_time"]["value"]
-                if (
-                    installations < minimum_installations
-                    or name in self.blocked_signatures
-                ):
+                if installations < minimum_installations or name in blocked_signatures:
```

Comment:
Why this change? If it's black formatting, could you do it directly on master with a separate commit and rebase this on top?

Reply:
Oh it's because of the removal of `self.`, forget my comment.